cmake/External/ck.cmake

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
follow spacing similar to other pytorch files. Dont use tabs.

Use lintrunner to verify and fix lint/NIT errors.